packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -387,6 +387,13 @@ export function createKusama (t: TFunction): EndpointOption {
           Dwellir: 'wss://shiden-rpc.dwellir.com'
         }
       },
+      {
+        info: 'shiden',
+        homepage: 'https://shiden.plasmnet.io/',
+        paraId: 2120,
+        text: t('rpc.kusama.shiden2', 'Shiden Crowdloan 2', { ns: 'apps-config' }),
+        providers: {}
```

Comment:
Lemme just test this one - not sure how it behaves with empty providers. (Normally there is at least 1 + isUnreachable). This is certainly valid, but just concerned that I missed a trick in the filtering.

Will check/approve/comment in the morning.